server/routerlicious/packages/gitresources/api-extractor-lint.json

```diff
@@ -0,0 +1,13 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "../../../../common/build/build-common/api-extractor-lint.json",
```

Comment:
We'll need to double check if this works, since the build for routerlicious runs inside a docker container where the folder structure is different/"incomplete" (not everything from the root of the repo exists there).

Reply:
For the record, I think we want to move away from building the server packages inside a container, but there's no scheduled work for it yet.

Reply:
Ah, okay. I wasn't aware. This apparently breaks the pnpm-patch I added as well. We may have to introduce local base configs for server then.

Reply:
@alexvy86 do you happen to know what controls which files get copied into the container prior to building? I'm seeing an issue where the pnpm patch I added can't be found when it gets around to running `pnpm i`, but the files exist under `server/routerlicious`?

Reply:
That said, I remembered you can also copy from the root of the repo with `COPY --from=root <source> <target>`, thanks to some work Tyler B did in the past. I think only the `server - routerlicious` pipeline can do that though, not historian nor gitrest (we can make easily make it available to them if needed). If routerlicious is the only one we need to update, you can do that to not duplicate the patches if you think it makes sense.

But on second thought, I think it's probably better to have them in the resource group?

Reply:
> But on second thought, I think it's probably better to have the patches in each resource group separately?

I think each group's `package.json` should patch separately, but it seems reasonable to me for them to reference the same patch file. I would prefer to not have duplicate copies of it if I can help it.

Reply:
But actually, I don't see a clean way to share the file (referenced by relative path) and make it work with both local installs and CI docker installs. So, I've kept the local patch copy for now.